clinica/iotools/converters/oasis3_to_bids/oasis3_to_bids.py

```diff
@@ -0,0 +1,165 @@
+# coding: utf8
+
+"""Convert OASIS dataset (http://www.oasis-brains.org/) to BIDS."""
+
+from clinica.iotools.abstract_converter import Converter
+from clinica.utils.stream import cprint
+
+
+class Oasis3ToBids(Converter):
+    def convert_clinical_data(self, clinical_data_dir, bids_dir):
+        """Convert the clinical data defined inside the clinical_specifications.xlx into BIDS.
+
+        Args:
+            clinical_data_dir: path to the folder with the original clinical data
+            bids_dir: path to the BIDS directory
+        """
+        from os import path
+        import os
+        import numpy as np
+        import clinica.iotools.bids_utils as bids
+        from clinica.utils.stream import cprint
+
+        cprint("Converting clinical data...")
+        bids_ids = bids.get_bids_subjs_list(bids_dir)
+
+        iotools_folder = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
+        clinic_specs_path = path.join(iotools_folder, "data", "clinical_specifications")
+
+        # -- Creation of modality agnostic files --
+        bids.write_modality_agnostic_files("OASIS-3", bids_dir)
+
+        # --Create participants.tsv--
+        participants_df = bids.create_participants_df(
+            "OASIS3", clinic_specs_path, clinical_data_dir, bids_ids
+        )
+
+        # Replace the values of the diagnosis_bl column
+        participants_df["diagnosis_bl"].replace([0.0, np.nan], "CN", inplace=True)
+        participants_df["diagnosis_bl"].replace(
+            [0.5, 1.0, 1.5, 2.0], "AD", inplace=True
+        )
+        # Following line has no sense
+        # participants_df['diagnosis_bl'].replace(participants_df['diagnosis_bl']>0.0, 'AD', inplace=True)
+        participants_df = participants_df.fillna("n/a")
+        participants_df.to_csv(
+            path.join(bids_dir, "participants.tsv"),
+            sep="\t",
+            index=False,
+            encoding="utf-8",
+        )
```

Comment:
Use the `na_rep` argument of `pandas.to_csv` instead.

Reply:
Should I replace in the same time all the "fillna" in clinica?

Reply:
No, let's keep the focus on your PR.